src/nodetreeview.cpp

```diff
@@ -230,7 +230,7 @@ void NodeTreeView::onUpdateAbsPath(const QString &oldPath, const QString &newPat
     });
 }
 
-void NodeTreeView::updateEditingIndex(const QPoint& pos)
+void NodeTreeView::updateEditingIndex(QPoint pos)
```

Comment:
What is your reasoning behind passing by value for these QPoint arguments? I know it's better to pass ints by value, but is this also the case for QPoint?

Reply:
QPoint is a "trivially copyable type" it's just a pair of ints, usually you don't pass ints by reference.